internal/cli/opsmanager/logs/decrypt.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/decryption"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type DecryptOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	cli.DownloaderOpts
+	inFileName                    string
+	kmipServerCAFileName          string
+	kmipClientCertificateFileName string
+	localKeyFileName              string
+}
+
+func (opts *DecryptOpts) initFiles() func() error {
+	// Validate the provided files can be open
+	return func() error {
+		return nil
+	}
+}
```

Comment:
[q] What's the purpose of this function?

Reply:
was a placeholder, I removed it, thanks